src/Components/Components/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,10 @@
+Compat issues with assembly Microsoft.AspNetCore.Components:
+CannotChangeAttribute : Attribute 'System.AttributeUsageAttribute' on 'Microsoft.AspNetCore.Components.BindElementAttribute' changed from '[AttributeUsageAttribute(4, AllowMultiple=true, Inherited=true)]' in the contract to '[AttributeUsageAttribute(AttributeTargets.Class, AllowMultiple=true, Inherited=true)]' in the implementation.
```

Comment:
Is it possible to add a blanket exclusion for decimal -> `enum` name pseudo-changes? Worst case would be a central list of cases to ignore, probably all involving `[AttributeUsage]` and `[EditorBrowsable]`.

Reply:
My suggestion is to create a single global exclusions file containing the unique lines from the many exclusion files in this PR. That should mean the exact `4` -> `AttributeTargets.Class` equivalents are listed. In particular, set `$(ApiCompatBaseline)` globally for all of the ref/ & src/ project pairs.

That said, adding options directly to `$(ApiCompatArgs)` may be more flexible i.e. there may be interesting options that help out.

Reply:
Ah well. The single baseline file won't work properly and would be a mess to maintain. The closest thing ApiCompat offers is `--exclude-attributes` and that would ignore the attributes completely rather than recognize `4 == AttributeTargets.Class` and so on.